Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -2038,4 +2011,34 @@
   <data name="Exclusions_AddExclusionDeclaration_DiscretionaryExclusionsLinkText" xml:space="preserve">
     <value>These exclusions are defined in &lt;a href="{0}" target="_blank"&gt;schedule 7 of the Procurement Act 2023 (opens in new tab)&lt;/a&gt;.</value>
   </data>
+  <data name="OrganisationRegistration_InternationalCountry_Heading" xml:space="preserve">
+    <value>Select which country your organisation is registered</value>
+  </data>
+  <data name="OrganisationRegistration_InternationalIdentifier_Country_Heading" xml:space="preserve">
+    <value>Select which country your organisation is registered</value>
+  </data>
```

Comment:
These two still seem to be duplicated. And still don't make gramatical sense?

`Select which country your organisation is registered` should be `Select which country your organisation is registered in`